src/functions/php-min-version.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/********************************************************************************
+ *
+ *
+ * IMPORTANT NOTE
+ *
+ * This file uses a global namespace since we will share it on all plugins
+ *
+ *
+ ********************************************************************************/
+
+// Only include these methods if they are not available already
+if ( ! function_exists( 'tribe_get_php_min_version' ) ) :
+
+/**
+ * Compares a given version to the required PHP version
+ *
+ * Normally we use Constant: PHP_VERSION
+ *
+ * @param  string  $version  Which PHP version we are checking against
+ *
+ * @since  TBD
+ *
+ * @return bool
+ */
+function tribe_is_not_min_php_version( $version = PHP_VERSION ) {
+	return version_compare( $version, tribe_get_php_min_version(), '<' );
+}
+
+/**
+ * Which is our required PHP min version
+ *
+ * @since  TBD
+ *
+ * @return string
+ */
+function tribe_get_php_min_version() {
+	return '5.6';
+}
+
+/**
+ * Returns the error message when php version min doesnt check
+ *
+ * @since  TBD
+ *
+ * @return string
+ */
+function tribe_not_php_version_message() {
+	$names = tribe_not_php_version_names();
+	$count_names = count( $names );
+	$last_connector = esc_html_x( ' and ', 'Plugin A "and" Plugin B', 'the-events-calendar' );
+	$many_connector = esc_html_x( ', ', 'Plugin A"," Plugin B', 'the-events-calendar' );
+
+	if ( 1 === $count_names ) {
+		$label_names = current( $names );
+	} elseif ( 2 === $count_names ) {
+		$label_names = current( $names ) . $last_connector . end( $names );
+	} else {
+		$last_name = array_pop( $names );
+		$label_names = implode( $many_connector, $names ) . $last_connector . $last_name;
+	}
+
+	return wp_kses_post( sprintf(
+			__( '<b>%1$s</b> requires <b>PHP %2$s</b> or higher.', 'the-events-calendar' ),
```

Comment:
Should this include a descriptive comment for translators that tells them what the variables are? I know we don't translate the plugin names or version numbers, but...
https://codex.wordpress.org/I18n_for_WordPress_Developers#Descriptions